app/views/user/documents/show.html.erb

```diff
@@ -13,7 +13,7 @@
   </div>
   <div class="card-footer">    
     <div class="row justify-content-center mb-4 mt-4">
-      <div class="col-sm-6">
+      <div class="col-sm-3">
         <%= render 'user/shares/modal_form', document: @document, classes: 'btn btn-primary btn-block' %>
```

Comment:
A better fix for this would be to remove `btn-block` from this button

Reply:
Nice! :+1: